pkg/unittest/input/capi/case-1-capa-mc.golden

```diff
@@ -0,0 +1,11 @@
+apiVersion: cluster.x-k8s.io/v1beta1
+kind: Cluster
+metadata:
+  name: test-installation
```

Comment:
This template is a CAPI MC

Reply:
I think if your code deserve some comments, they should rather be in the code than in the PR :stuck_out_tongue_closed_eyes:

Reply:
Well I was trying to explain that to you but yes you're right, I'll make this more readable first